stripe/src/main/java/com/stripe/android/ErrorParser.java

```diff
@@ -31,12 +29,12 @@ static StripeError parseError(String rawError) {
         try {
             JSONObject jsonError = new JSONObject(rawError);
             JSONObject errorObject = jsonError.getJSONObject(FIELD_ERROR);
-            stripeError.charge = StripeJsonUtils.optString(errorObject, FIELD_CHARGE);
-            stripeError.code = StripeJsonUtils.optString(errorObject, FIELD_CODE);
-            stripeError.decline_code = StripeJsonUtils.optString(errorObject, FIELD_DECLINE_CODE);
-            stripeError.message = StripeJsonUtils.optString(errorObject, FIELD_MESSAGE);
-            stripeError.param = StripeJsonUtils.optString(errorObject, FIELD_PARAM);
-            stripeError.type = StripeJsonUtils.optString(errorObject, FIELD_TYPE);
+            stripeError.charge = errorObject.optString(FIELD_CHARGE);
+            stripeError.code = errorObject.optString(FIELD_CODE);
+            stripeError.decline_code = errorObject.optString(FIELD_DECLINE_CODE);
+            stripeError.message = errorObject.optString(FIELD_MESSAGE);
+            stripeError.param = errorObject.optString(FIELD_PARAM);
+            stripeError.type = errorObject.optString(FIELD_TYPE);
```

Comment:
`optString` has actually always been build into `JSONObject`. I was previously using a utility function here that does one thing that frustrates both the Android models and our servers.

If optString (string in particular) can't find the value requested, it doesn't return null -- it returns the empty string. I used to have tons of bugs in the models dealing with this, and why the utility function `optString` (which moved over to the models directory with the rest of StripeJsonUtils) is still in use, and basically corrects this by returning null if the value isn't found.